server.py

```diff
@@ -50,6 +51,67 @@ def get_shoes(self):
         conn.close()
         return jsonify(data)
     
+    def post_shoes(self):
+        conn = self.__get_db_connection()
+        cursor = conn.cursor()
+
+        #     cursor.execute('''
+        #     CREATE TABLE IF NOT EXISTS shoes (
+        #         shoes_id INTEGER PRIMARY KEY,
+        #         brand TEXT NOT NULL,
+        #         shoe_name TEXT NOT NULL,
+        #         shape TEXT NOT NULL,
+        #         size INT NOT NULL,
+        #         menwear TEXT NOT NULL,
+        #         womanwear TEX NOT NULL,
+        #         price INT NOT NULL,
+        #         manufacturer TEXT NOT NULL      
+        #     )
+        # ''')
+        data = request.get_json()
+        #print(data)
+        brand = data['brand']
```

Comment:
Instead of creating this new variable, you can directly use it in the _checklist_ dict like this:
_checklist={"brand": data["brand"]}_